app/views/posts/show.html.erb

```diff
@@ -1 +1,26 @@
-<p> Here is a list of all posts: </p>
\ No newline at end of file
+<% begin %>
+  Here is a list of post by user ID:
+<% end %>
+
+<div class="comment-wrapper">
+  <div class="comment-head">
+    <h1 class="post-title">
+      <%= @post.title %> By <%= @post.author.name %>
+    </h1>
+    <div class="comments-likes">
+      <p>comments: <%= @post.comments_counter %></p>
+      <p>likes: <%= @post.likes_counter %></p>
+    </div>
+  </div>
+
+  <p><%= @post.text %></p>
+
+  <div class="comments">
+    <% @comments.each do |comment| %>
+      <div class="single-comment">
+        <span><%= comment.author.name %> : <%= comment.text %></span>
+      </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] Kindly make sure you stick to the original template

### Your design :point_down: 
![image](https://github.com/saluumaa/blog-app/assets/97100091/2edbb8a4-1fd2-488f-b56f-38b153330c4a)

### Original template :point_down: 
![image](https://github.com/saluumaa/blog-app/assets/97100091/efb17690-8953-401c-ab03-acd6a541264e)